src/dom/__tests__/errorHook.test.ts

```diff
@@ -142,51 +142,4 @@ describe('error hook tests', () => {
       expect(result.error).toBe(undefined)
     })
   })
-
-  describe('error output suppression', () => {
```

Comment:
this moved to [a new file](https://github.com/testing-library/react-hooks-testing-library/pull/624/files#diff-8bbf97dce934ce64ecc78bc41856f39d3fbdeca3cfd48ab25f21555649e1a677)